spec/latest/core/core.jsonld

```diff
@@ -98,23 +98,23 @@
       "subClassOf": "rdfs:Resource",
       "label": "Hydra Resource",
       "comment": "The class of dereferenceable resources.",
-      "vs:term_status": "testing"
+      "vs:term_status": "stable"

```

Comment:
All of those changes appear unrelated to the matter at hand IMO

Reply:
Ok. I didn't want to create a separate PR that is unrelated to any issue, but let it be that way. I'll update it ASAP.